recipe-hooks-project/src/PrivateRoutes.js

```diff
@@ -0,0 +1,41 @@
+import React, {useContext} from "react";
+import {Redirect, Route} from "react-router-dom";
+import AppPage from "./AppPage";
+import Home from "./Home";
+import Register from "./Register";
+import Profile from "./Profile";
+import Ingredients from "./Ingredients";
+import Tags from "./Tags";
+import Recipes from "./Recipes";
+import RecipeDetails from "./RecipeDetails";
+import {AuthContext} from "./contexts/AuthenticationProvider";
+
+export function PrivateRoutes() {
```

Comment:
The file is called PrivateRoutes but it actually contains the public too, why not calling it just `Routes`?